provisioning/aws/scripts/deploy.sh

```diff
@@ -2,10 +2,16 @@
 set -Eeuo pipefail
 
 aws eks update-kubeconfig --name "$AWS_EKS_CLUSTER_NAME" --region "$AWS_REGION"
-kubectl config set-context --current --namespace default
+kubectl config set-context --current --namespace templates-api
```

Comment:
Tohle celý se může podle mě vynechat, všechny kubernetes manifesty v sobě manifest mají explicitně uvedený. Zkusím to teď deploynout bez toho

Reply:
jj maju, bolo to tu povodne, tak som to nemazal